app/config/messaging.js

```diff
@@ -0,0 +1,45 @@
+import Joi from 'joi'
```

Comment:
Would say this message config needs updated to use Convict instead of Joi since this was agreed in the Tech Huddle. Example can be found [here](https://github.com/DEFRA/fcp-fd-comms/blob/main/app/config/messaging.js). Same applied for the database config if you've also used Joi?

Reply:
We have decide to push this onto tech debt.